components/experiments/ExperimentHeader.vue

```diff
@@ -14,23 +14,21 @@
           <path fill="#999" d="M10.803 15.047l-2.121 2.121L.197 8.683l2.121-2.121z" />
         </svg>
         Back to experiments
-      </nuxt-link>
+      </AppLink>
       <h1>{{ name }}</h1>
       <p class="experiment-header__meta">
         <span class="experiment-header__author">
           {{ authors }}
         </span>
         |
-        <a
+        <AppLink
           class="experiment-header__source-code-link"
-          :href="source"
-          target="_blank"
-          rel="noopener"
+          :url="source"
         >
           Explore the code
-        </a>
+        </AppLink>
       </p>
-      <Cta v-if="launch" :to="launch">
+      <Cta v-if="launch" :link="{ url: launch }">
```

Comment:
I think you forgot to change this since `link` property no longer exists.

Reply:
The ExperimentHeader component has been deleted